src/encoded/tests/test_audit_experiment.py

```diff
@@ -2915,3 +2924,43 @@ def test_audit_experiment_with_biosample_not_missing_nih_consent(testapp, experi
         ]
     )
 
+
+def test_is_matching_biosample_control(testapp, biosample, ctrl_experiment, treatment_time_series):
+    from encoded.audit.experiment import is_matching_biosample_control
+    exp = testapp.get(ctrl_experiment['@id'] + '@@index-data')
```

Comment:
Is there a reason for using the `@@index-data` view instead of `@@embedded` view? Didn't realized `@@index-data` existed but neat!